src/core/grid_based_algorithms/lbgpu_cuda.cu

```diff
@@ -2181,9 +2150,11 @@ __global__ void reset_boundaries(LB_boundaries_gpu boundaries) {
  *  @param[out]    n_b     Local node residing in array b
  *  @param[in,out] d_v     Local device values
  *  @param[in,out] node_f  Local node force density
+ *  @param[in]     ek_parameters_gpu  Parameters for the electrokinetics
  */
 __global__ void integrate(LB_nodes_gpu n_a, LB_nodes_gpu n_b, LB_rho_v_gpu *d_v,
                           LB_node_force_density_gpu node_f,
+                          EK_parameters *ek_parameters_gpu,
```

Comment:
Here e.g.

Reply:
this was intended by this PR, right?

Reply:
No, those are not needed and were removed in the master after this branch was forked of